### PR TITLE
Adding hackathon blog post

### PR DESCRIPTION
**Pull request by dnoegel**

### CODE REVIEW

File: source/blog/_posts/2016-02-11-hackathon-2016-1.md [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+
+<div style="clear:both"></div>
+
+## Light it up			
+
+**Team**: Stephan Pohl<br>
+**Project**: Continuous integration visualisation   
+
+Continuous integration is a key factor for high quality software. It ensures a continuous quality throughout the software.
+On the other hand developers have to be aware of the current status of the project in a non-disturbing way matter.
```
Comment (klarstil) [inline source/blog/_posts/2016-02-11-hackathon-2016-1.md]: @dnoegel You can remove the "matter" e.g.

"On the other hand developers have to be aware of the current status of the project in a non-disturbing way ~~matter~~".

File: source/blog/_posts/2016-02-11-hackathon-2016-1.md [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+If it's a known error to us, there might also be a suggestion on how to fix that special issue or how that issue may occur.
+
+Additionally the plugin should provide some helpful functions, e.g. deactivating all 3rd-party plugins and re-activating
+them later or simulating the error, that just occurred.
+
+<div style="clear:both"></div>
+	
+## Syncing down the house		
+
+**Team**: Jan Philipp Pietzrzyk<br>
```
Comment (JanPietrzyk) [inline source/blog/_posts/2016-02-11-hackathon-2016-1.md]: Pietrzyk please

Comment (JanPietrzyk) [inline source/blog/_posts/2016-02-11-hackathon-2016-1.md]: It's that simple ;)

Comment (dnoegel) [inline source/blog/_posts/2016-02-11-hackathon-2016-1.md]: Pietrzazikizyk?

**Pull request merged**